azure-templates/steps-diff.yml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  - name: lvVersionToDiff
+    type: string
+
+# Note: the following variables are defined in pre-job-steps.yml:
+  # customDeviceRepoName
+
+steps:
+  - powershell: |
+      pip install virtualenv
+      virtualenv env
+      .\env\Scripts\activate
+      pip install requests
+    displayName: Prepare diff environment
+  - checkout: self
+    fetchDepth: 0
+    displayName: Fetch full repo
+  - powershell: |
+      git clone https://github.com/ni/niveristand-custom-device-build-tools.git `
+      $(customDeviceRepoName)\niveristand-custom-device-build-tools
+    displayName: Check out NI VS Custom Device Build Tools repository
+  - powershell: |
+      .\env\Scripts\activate
+      cd $(customDeviceRepoName)
+      $currentDir = Get-Location
+      If (Test-Path $currentDir/diff_dir)
+      {
+        Remove-Item $currentDir/diff_dir -Recurse
+      }
+      New-Item -Path $currentDir -Name "diff_dir" -ItemType "directory"
+      $targetBranch = "$(System.PullRequest.TargetBranch)"
+      $resourcesDir = "niveristand-custom-device-build-tools/resources"
+      python -u $resourcesDir/labview_diff.py `
+      "$currentDir" "$currentDir/diff_dir" "${{parameters.lvVersionToDiff}}" `
```

Comment:
if using ` to extend a command to multiple lines I have preferred indentation to make it more clear that this is a continuation of a previous command